app/controllers/rails/view_components_controller.rb

```diff
@@ -16,23 +16,31 @@ class Rails::ViewComponentsController < Rails::ApplicationController # :nodoc:
   def index
     @previews = ViewComponent::Preview.all
     @page_title = "Component Previews"
+    opts = default_preview_layout.nil? ? {} : { layout: default_preview_layout }
+    prepend_application_view_paths if opts.key?(:layout)
     # rubocop:disable GitHub/RailsControllerRenderPathsExist
-    render "components/index"
+    render "components/index", **opts
     # rubocop:enable GitHub/RailsControllerRenderPathsExist
   end
 
   def previews
     if params[:path] == @preview.preview_name
       @page_title = "Component Previews for #{@preview.preview_name}"
+      opts = default_preview_layout.nil? ? {} : { layout: default_preview_layout }
+      prepend_application_view_paths if opts.key?(:layout)
       # rubocop:disable GitHub/RailsControllerRenderPathsExist
-      render "components/previews"
+      render "components/previews", **opts
       # rubocop:enable GitHub/RailsControllerRenderPathsExist
     else
       prepend_application_view_paths
       @example_name = File.basename(params[:path])
       @render_args = @preview.render_args(@example_name)
       layout = @render_args[:layout]
-      opts = layout.nil? ? {} : { layout: layout }
+      opts = if layout.nil?
+               default_preview_layout.nil? ? {} : { layout: default_preview_layout }
+             else
+               { layout: layout }
+             end
```

Comment:
Functionally this is great but stylistically nested if conditions like this feel a little hard to read. It would be a little clearer as 
```suggestion
      layout_override = layout.presence? || default_preview_layout.presence?
      opts = layout_override.nil? ? {} : { layout: layout_override }
```

Do we have a test case where layout = false and default_preview_layout is set? Important that we're getting that right by checking nil? presence? etc

Reply:
Similar comment as above about explicitly setting the layout rather than using `prepend_application_view_paths`

Reply:
> Functionally this is great but stylistically nested if conditions like this feel a little hard to read. It would be a little clearer as

I'm fine with cleaning that up a bit, but that syntax change didn't pass the a couple tests.

> Do we have a test case where layout = false and default_preview_layout is set? Important that we're getting that right by checking nil? presence? etc

Good call, adding a test to check false and switching logic from .nil? to .blank? did the trick.